docs/DeveloperGuide.md

```diff
@@ -997,6 +1089,25 @@ testers are expected to do more *exploratory* testing.
 
 ---
 
+### Suggesting a show
+
+1. Prerequisites: None.
+2. Launch the app.
+3. Command: `suggest`
+  1. Test case:
+    1. Condition: Displayed list of show is empty.
+    2. Expected: Trackermon returns an error message informing user that displayed list of show is empty.
+  2. Test case:
```

Comment:
In our manual test cases, we should follow the format used, where it is `Test case: ` followed by the test case. In this case, it would be `Test case: Displayed list of shows only contains one show.`

Reply:
Hmm, isn't this kind of a condition?

Reply:
I was of the mind that the Test case was testing for when the displayed list of shows is empty. Then I'd place the condition next to the test case as per my suggestion haha.